build.sbt

```diff
@@ -125,100 +86,20 @@ def circeCrossModule(path: String, crossType: CrossType = CrossType.Full) = {
     )
 }
 
-lazy val docsMappingsAPIDir =
-  settingKey[String]("Name of subdirectory in site target directory for api docs")
-
-lazy val docSettings = allSettings ++ Seq(
-  micrositeName := "circe",
-  micrositeDescription := "A JSON library for Scala powered by Cats",
-  micrositeAuthor := "Travis Brown",
-  micrositeHighlightTheme := "atom-one-light",
-  micrositeHomepage := "https://circe.github.io/circe/",
-  micrositeBaseUrl := "/circe",
-  micrositeDocumentationUrl := s"${docsMappingsAPIDir.value}/io/circe",
-  micrositeDocumentationLabelDescription := "API Documentation",
-  micrositeGithubOwner := "circe",
-  micrositeGithubRepo := "circe",
-  micrositeExtraMdFiles := Map(
-    file("CONTRIBUTING.md") -> ExtraMdFileConfig(
-      "contributing.md",
-      "docs",
-      Map("title" -> "Contributing", "position" -> "6")
-    )
-  ),
-  micrositeExtraMdFilesOutput := resourceManaged.value / "main" / "jekyll",
-  micrositeTheme := "pattern",
-  micrositePalette := Map(
-    "brand-primary" -> "#5B5988",
-    "brand-secondary" -> "#292E53",
-    "brand-tertiary" -> "#222749",
-    "gray-dark" -> "#49494B",
-    "gray" -> "#7B7B7E",
-    "gray-light" -> "#E5E5E6",
-    "gray-lighter" -> "#F4F3F4",
-    "white-color" -> "#FFFFFF"
-  ),
-  micrositeConfigYaml := ConfigYml(yamlInline = s"""
-      |scalafiddle:
-      |  dependency: io.circe %%% circe-core % $scalaFiddleCirceVersion,io.circe %%% circe-generic % $scalaFiddleCirceVersion,io.circe %%% circe-parser % $scalaFiddleCirceVersion
-    """.stripMargin),
-  docsMappingsAPIDir := "api",
-  addMappingsToSiteDir(ScalaUnidoc / packageDoc / mappings, docsMappingsAPIDir),
-  ghpagesNoJekyll := true,
-  ScalaUnidoc / unidoc / scalacOptions ++= Seq(
-    "-groups",
-    "-implicits",
-    "-skip-packages",
-    "scalaz",
-    "-doc-source-url",
-    scmInfo.value.get.browseUrl + "/tree/master€{FILE_PATH}.scala",
-    "-sourcepath",
-    (LocalRootProject / baseDirectory).value.getAbsolutePath
-  ),
-  /* Publish GitHub Pages { */
-  gitHubPagesOrgName := "circe",
-  gitHubPagesRepoName := "circe",
-  gitHubPagesSiteDir := baseDirectory.value / "target" / "site",
-  /* } Publish GitHub Pages */
-  scalacOptions ~= {
-    _.filterNot(Set("-Yno-predef"))
-  },
-  git.remoteRepo := "git@github.com:circe/circe.git",
-  ScalaUnidoc / unidoc / unidocProjectFilter := inProjects(
-    numbers.jvm,
```

Comment:
Instead of deleting this unidoc stuff, let's move it into a published unidoc artifact.
https://typelevel.org/sbt-typelevel/site.html#how-can-i-publish-unidoc-api-docs